vbr/vbrCore/functions/el_ModUnrlx_dTdP_f.m

```diff
@@ -28,11 +28,25 @@
   dT = (VBR.in.SV.T_K-T_K_ref);
   dP = (VBR.in.SV.P_GPa*1e9 - P_Pa_ref);
 
-  % calculate shear modulus at T,P of interest
-  Gu_TP = calc_Gu(Gu_0,dT,dP,dG_dT0,dG_dP0);
+  if isfield(VBR.in.elastic,'Gu_TP') && isfield(VBR.in.elastic,'Ku_TP')
+    % Load unrelaxed shear and bulk moduli (at T,P of interest)
+    Gu_TP = VBR.in.elastic.Gu_TP; % Pa
+    Ku_TP = VBR.in.elastic.Ku_TP; % Pa
+    
+  elseif isfield(VBR.in.elastic,'Gu_TP') && ~isfield(VBR.in.elastic,'Ku_TP')
+    % Load unrelaxed shear modulus (at T,P of interest)
+    Gu_TP = VBR.in.elastic.Gu_TP; % Pa
+    
+    % calculate bulk modulus
+    warning(['No Bulk Modulus found. Calculating assuming nu=',num2str(nu)]);
```

Comment:
one more change... i'd prefer to not use a real warning here, since I see this as more of an informational message. 
```suggestion
disp(['Unrelaxed shear modulus was provided without an unrelaxed bulk modulus.', ...
          ' Calculating bulk modulus assuming nu=', num2str(nu), ...
          '. Set VBR.in.elastic.Ku_TP to specify a value.']);
```